documentation/docs/11-ssr-and-javascript.md

```diff
@@ -0,0 +1,77 @@
+
+You can disable SSR app-wide with the [`ssr` config option](#configuration-ssr), or a page-level `ssr` export:
+
+```html
+<script context="module">
+	export const ssr = false;
+</script>
+```
+
+### router
+
+SvelteKit includes a client-side router that intercepts navigations (from the user clicking on links, or interacting with the back/forward buttons) and updates the page contents, rather than letting the browser handle the navigation by reloading.
+
+In certain circumstances you might need to disable this behaviour with the app-wide [`router` config option](#configuration-router) or the page-level `router` export:
+
+```html
+<script context="module">
+	export const router = false;
+</script>
+```
+
+### hydrate
+
+Ordinarily, SvelteKit 'hydrates' your server-rendered HTML into an interactive page. Some pages don't require JavaScript at all — many blog posts and 'about' pages fall into this category. In these cases you can skip hydration when the app boots up with the app-wide [`hydrate` config option](#configuration-hydrate) or the page-level `hydrate` export:
+
+```html
+<script context="module">
+	export const hydrate = false;
+</script>
+```
+
+> If `hydrate` and `router` are both `false`, SvelteKit will not add any JavaScript to the page at all.
+
+### prerender
+
+It's likely that at least some pages of your app can be represented as a simple HTML file, since they contain no dynamic or user-specific data. These pages can be _prerendered_ by your [adapter](#adapters).
+
+If your entire app is suitable for prerendering, you could use [`adapter-static`](https://github.com/sveltejs/kit/tree/master/packages/adapter-static), which will generate HTML files for every page, plus additional files that are requested by `load` functions in those pages.
+
+More likely, you'll only want to prerender specific pages in your app. You'll need to annotate these pages:
+
+```html
+<script context="module">
+	export const prerender = true;
+</script>
+```
+
+The prerenderer will start at the root of your app and generate HTML for any prerenderable pages it finds. Each page is scanned for `<a>` elements that point to other pages that are candidates for prerendering — because of this, you generally don't need to specify which pages should be accessed. If you _do_ need to specify which pages should be accessed by the prerenderer, you can do so with the `pages` option in the [prerender configuration](#configuration-prerender).
+
+#### When not to prerender
+
+The basic rule is this: for a page to be prerenderable, any two users hitting the same page of your app must get the same content from the server. In other words, any app that involves user sessions or authentication is _not_ a candidate for the static adapter.
```

Comment:
I think this is worded a little confusingly. Do we want to be talking about prerendering particular pages, or the whole app?

Reply:
This whole section was copied from the old `Prerendering` section, and it is indeed a bit confusing. Have clarified it, hopefully

Reply:
Yep, sounds better now :+1: